app/config/security.yml

```diff
@@ -79,6 +79,7 @@ security:
         - { path: ^/administration/login-check, role: IS_AUTHENTICATED_ANONYMOUSLY }
         - { path: "/administration.*", role: ROLE_ADMINISTRATION_ACCESS }
 
+        - { path: "/account/orders/[0-9-]*", role: IS_AUTHENTICATED_ANONYMOUSLY }
```

Comment:
I don't think we want to do that... Maybe add other route for that case? (we could reuse the controller action)